src/class/PollManager/PollManager.hpp

```diff
@@ -0,0 +1,50 @@
+
+#ifndef POLL_MANAGER_HPP
+#define POLL_MANAGER_HPP
+
+#include <algorithm>
+#include <fcntl.h>
+#include <iostream>
+#include <poll.h>
+#include <string>
+#include <vector>
+
+#ifdef TEST_ON
+#define private public
+#define protected public
+#endif
+
+class PollManager
```

Comment:
i also created a version of this class.
after we merge this we will need to combine them since each one of us was covering different flows